bincrafters_envy/appveyor.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+
+from __future__ import print_function
+import json
+
+from .base import Base
+
+
+class Appveyor(Base):
+    name = "appveyor"
+    default_host = 'https://ci.appveyor.com'
+
+    def __init__(self, config, host):
+        super(Appveyor, self).__init__(config, host)
+        self._headers = {
+            'Authorization': 'Bearer {token}'.format(token=self._token),
+            'Content-type': 'application/json'
+        }
+        self._account = self._read_account(config) or 'BinCrafters'
+        if self._token.startswith("v2.0"):
+            self._endpoint = "{host}/api/account/{account}".format(host=self._host, account=self._account)
+        else:
+            self._endpoint = "{host}/api".format(host=self._host)
+        self._github_account = self._read_account(config, "github") or 'bincrafters'
+
+    def list(self):
+        projects = self._get(url='/projects')
+        projects = [project['slug'] for project in projects]
+        return projects
+
+    def _project_url(self, project_slug):
+        return '/projects/%s/%s' % (self._account, project_slug)
+
+    def exists(self, project_slug):
+        appveyor_url = self._project_url(project_slug)
+        try:
+            self._get(url=appveyor_url)
+            return True
+        except Exception:
+            return False
+
+    def add_one(self, project_slug):
+        repository_name = '{accountName}/{projectSlug}'.format(
+            accountName=self._github_account,
+            projectSlug=project_slug
+        )
+        request = {
+            'repositoryProvider': 'gitHub',
+            'repositoryName': repository_name
+        }
+        self._post(url='/projects', data=json.dumps(request))
```

Comment:
```suggestion
        self._post(url='/projects', json=request)
```
So you can removed `Application/content-Type` from headers